Src/Generated/Google.Apis.Discovery.v1/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.15.0",
+
+  "dependencies": {
+    "GoogleApis_dotnetcore": { "target": "project" },
```

Comment:
What's the underscore doing here? My guess is that this is part of the build I'm not aware of...